api-tests/secure-pipeline/run-tests.sh

```diff
@@ -12,7 +12,7 @@ export CORE_BACK_INTERNAL_API_KEY
 
 cd /api-tests
 
-npm run test:build
+npm run test:build -- --profile codepipeline
```

Comment:
Had to look up what was going on here: https://github.com/cucumber/cucumber-js/blob/main/docs/profiles.md

Neat!